interfaces/lego-boost-sse/README.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: doc
+title: Lego Boost Kinetic AR Prototyping
+permalink: /docs/vuforia-spatial-robotic-addon/interfaces/lego-boost/README
+---
+
+> IMPORTANT NOTE: The accuracy of this interface highly depends on floor friction and motor power that can have small variations from robot to robot. In order to improve the accuracy of the interface make sure you adjust the parameters properly for your own Lego Boost.
+
+## Lego Boost Kinetic AR Prototyping
+
+![Image of VST tool](../../resources/gifs/boost.gif)
+
+This is a reference to build a [**Vuforia Spatial Toolbox** (VST)](https://forum.spatialtoolbox.vuforia.com ) path planning prototype using robotic interfaces for a Lego Boost robot. 
+This is based on our [**Kinetic AR** UX framework](https://dl.acm.org/doi/pdf/10.1145/3334480.3382814) and it works with two components:
+* **[Robotic Addon Interface](../../)** [To connect to the robot hardware via BLE]
+* **Motion Tool** [AR UI when looking at the robot with your mobile device]
+
+It allows a connection to the robot and a spatial synchronization 
+between the robot coordinate system and the Vuforia groundplane coordinate system. 
+It also allows for path planning.
+
+### Getting Started
+
+These instructions will explain how to setup the Lego Boost robotic addons and tools. 
+You will have to clone or download the code for the VST repositories as explained later.
+
+#### Prerequisites
+
+* A Lego Boost robot running on 2 wheels (either tank or regular)
+* A laptop to be able to run the Vuforia Spatial Edge Server
+* A mobile phone with the Vuforia Spatial Toolbox application installed
+* You will also need the [Vuforia](https://developer.vuforia.com/) object target or image target to identify the robot with the mobile device [Later explained]
+
+### Download code
+
+1. Download the [vuforia-spatial-edge-server](https://github.com/ptcrealitylab/vuforia-spatial-edge-server) code and place it in your Development folder on your computer.
+
+2. Download the [vuforia-spatial-robotic-addon](https://github.com/ptcrealitylab/vuforia-spatial-robotic-addon) code and place it in the following folder:
```

Comment:
Do all the references to "vuforia-spatial-robotic-addon" need to be updated to "vuforia-spatial-motion-addon" now that we've moved this to a different repo?

Reply:
Yep. will update all of this and create a new readme in the root